git_machete/client.py

```diff
@@ -825,31 +832,35 @@ def traverse(
 
             if needs_remote_sync:
                 any_action_suggested = True
+                annotation_no_push_qualifier = None
```

Comment:
The implementation seems inconsistent with what's above around rebase: `rebase` vs `annotation_no_push_qualifier` boolean vars 🤔